catalog/app/containers/Bucket/PackageUpdateDialog.js

```diff
@@ -209,80 +634,38 @@ function FilesInput({ input, meta, uploads, setUploads, errors = {} }) {
     'Drop files here or click to browse'
   )
 
-  const pipeValue = useMemoEq([onInputChange, value], () => (...fns) =>
-    R.pipe(...fns, onInputChange)(value),
-  )
+  const ref = React.useRef()
```

Comment:
Probably name `state` instead of `ref` has more sense

Reply:
```
const state = useRef()
state.current = { … }

dispatch = (action) => {
  const currentState = state.current
  if (currentState.disabled) return
  …
}
```

Reply:
i think it may be confusing, bc "state" has some specific meaning in the react context (useState and re-rendering on change)